temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -125,4 +125,49 @@ service CloudService {
             get: "/api/v1/cloud/regions/{region}",
         };
     }
+
+    // Get all groups
+    rpc GetGroups (GetGroupsRequest) returns (GetGroupsResponse) {
+        option (google.api.http) = {
+            get: "/api/v1/groups",
```

Comment:
```suggestion
            get: "/api/v1/cloud/groups",
```

We now prefix control plane with common HTTP prefix so it can live side-by-side with dataplane on the same single endpoint (also change this to `user-groups` if we change the RPC name)